src/containerapp/azext_containerapp/commands.py

```diff
@@ -262,3 +262,9 @@ def load_command_table(self, args):
         g.custom_command('set', 'create_or_update_java_logger', supports_no_wait=True)
         g.custom_command('delete', 'delete_java_logger', supports_no_wait=True)
         g.custom_show_command('show', 'show_java_logger')
+
+    with self.command_group('containerapp env maintenanceconfig') as g:
```

Comment:
what about `az containerapp env maintenance-configuration`?
Is it possible to use a shorter name like  `az containerapp env maintenance-config` ?
Refer existing command:
![image](https://github.com/user-attachments/assets/a9eb35f1-2bc1-45ec-ba3d-36c3f9b5f9d8)